modules/osm/tags.js

```diff
@@ -65,6 +65,15 @@ export var osmAreaKeysExceptions = {
     }
 };
 
+/** iD will not allow nodes with this tag to be moved */
+export const osmTagsPreventingMove = {
+    man_made: {
+        survey_point: true
+    },
+    latitude: true,
+    longitude: true,
```

Comment:
I'm not sure if encouraging these two keys is a good idea, since most appear to be introduced by bodged imports

Reply:
Yeah, I could see this legitimizing the two keys as a “one weird trick” for locking a feature. Imagine someone pranking other mappers by adding a `longitude` to all the POIs they map. It would never show up in the fields, only the raw tags, hidden by default.